nightly/trigger-authorino-standalone-cronjob.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: trigger-nightly-authorino-standalone
+spec:
+  schedule: "0 6 * * *"
+  concurrencyPolicy: Forbid
+  startingDeadlineSeconds: 300
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: pipeline
+          containers:
+            - name: trigger-nightly-authorino-standalone
+              image: curlimages/curl
```

Comment:
uses image from `docker.io`. I would change them to `quay.io/curl/curl:latest` to prevent being rate-limitted by registry

Reply:
I'll change it in the next PR